proxy/pkg/user/backend/accounts_test.go

```diff
@@ -108,13 +108,13 @@ func TestAuthenticateFailed(t *testing.T) {
 func TestCreateUserFromClaims(t *testing.T) {
 	exp := mockAccResp[0]
 	accBackend := newAccountsBackend([]*accounts.Account{}, expectedRoles)
-	act, _ := accBackend.CreateUserFromClaims(context.Background(), &oidc.StandardClaims{
-		DisplayName:       mockAccResp[0].DisplayName,
-		PreferredUsername: mockAccResp[0].OnPremisesSamAccountName,
-		Email:             mockAccResp[0].Mail,
-		UIDNumber:         "1",
-		GIDNumber:         "2",
-		Groups:            []string{"g1", "g2"},
+	act, _ := accBackend.CreateUserFromClaims(context.Background(), map[string]interface{}{
+		oidc.Name:       mockAccResp[0].DisplayName,
```

Comment:
There is no `display_name` claim in the spec: https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims only name, but the idp may send a custom claim.
- [ ] fallback to `display_name`? or `displayname`? configurable?

Reply:
is done in accounts.go